packages/sodium/src/js/modules/SessionData.js

```diff
@@ -161,6 +162,7 @@ export default class SessionData {
       mainVideo = await this.waitMainVideo();
 
       console.log(`VIDEOMARK: STATE CHANGE found main video ${mainVideo.get_video_id()}`);
+      this.location = window.location.href;
```

Comment:
文字列として参照している箇所よりURLオブジェクトとして参照している箇所のほうが多いようなので

```suggestion
      this.location = new URL(window.location.href);
```

Reply:
URLオブジェクトに変更しました。